cpp/include/raft/neighbors/ivf_pq_types.hpp

```diff
@@ -161,9 +161,9 @@ constexpr static uint32_t kIndexGroupVecLen = 16;
 template <typename IdxT>
 constexpr static IdxT kOutOfBoundsRecord = std::numeric_limits<IdxT>::max();
 
-template <typename SizeT = uint32_t>
+template <typename ValueT = uint8_t, typename SizeT = uint32_t>
```

Comment:
Do we need to have `ValueT` as a template parameter, when in practice we always want it to be uint8? Unlike IVF-FLAT, where the value type is related to the type of the dataset, here in IVF-PQ we compress the dataset to a set of bits,  and the value_type is just used to access memory with 1 byte granularity).

Reply:
Yes, I'd prefer to have this reverted along with the changes to ivf_list_types.cuh unless there is a very strong reason for this change.